<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastDataLoggerDLLRF</class>
 <widget class="TaurusWidget" name="FastDataLoggerDLLRF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ctrffdl</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QScrollArea" name="generalScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>786</width>
        <height>986</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load Data</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>356</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="timeAndDecimation" name="timeAndDecimation"/>
         </item>
         <item>
          <widget class="QPushButton" name="replotButton">
           <property name="text">
            <string>Replot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="plotsTab">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="loops1Tab">
          <attribute name="title">
           <string>Loops1</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QScrollArea" name="loops1ScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="loops1ScrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>756</width>
                <height>818</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="Loops" name="loops1Plots"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="loops2Tab">
          <attribute name="title">
           <string>Loops2</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QScrollArea" name="loops2ScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="loops2ScrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>620</width>
                <height>624</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="Loops" name="loops2Plots"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="diagTab">
          <attribute name="title">
           <string>Diagnostics</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QScrollArea" name="diagScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="diagScrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>620</width>
                <height>828</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="Diagnostics" name="diagnosticsPlots"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Diagnostics</class>
   <extends>TaurusWidget</extends>
   <header>diagnostics</header>
  </customwidget>
  <customwidget>
   <class>Loops</class>
   <extends>TaurusWidget</extends>
   <header>loops</header>
  </customwidget>
  <customwidget>
   <class>timeAndDecimation</class>
   <extends>TaurusWidget</extends>
   <header>timeanddecimation</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>loadButton</tabstop>
  <tabstop>replotButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>plotsTab</tabstop>
  <tabstop>loops1ScrollArea</tabstop>
  <tabstop>loops2ScrollArea</tabstop>
  <tabstop>diagScrollArea</tabstop>
  <tabstop>generalScrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
